content/language/php/run-tests.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Run PHP tests in a container
+keywords: php, test
+description: Learn how to run your PHP tests in a container.
+---
+
+## Prerequisites
+
+Complete all the previous sections of this guide, starting with [Containerize a PHP application](containerize.md).
+
+## Overview
+
+Testing is an essential part of modern software development. Testing can mean a
+lot of things to different development teams. There are unit tests, integration
+tests and end-to-end testing. In this guide you take a look at running your unit
+tests in Docker when developing and when building.
+
+## Run tests when developing locally
+
+The sample application already has a PHPUnit test inside the `tests` directory. When developing locally, you can use Compose to run your tests.
+
+Run the following command in the `docker-php-sample` directory to run the tests inside a container.
+
+```console
+$ docker compose run --build --rm server ./vendor/bin/phpunit tests/HelloWorldTest.php
+```
+
+You should see output that contains the following.
+
+```console
+Hello, Docker!PHPUnit 9.6.13 by Sebastian Bergmann and contributors.
+
+.                                                                   1 / 1 (100%)
+
+Time: 00:00.003, Memory: 4.00 MB
+
+OK (1 test, 1 assertion)
+```
+
+To learn more about the command, see [docker compose run](/engine/reference/commandline/compose_run/).
+
+## Run tests when building
+
+To run your tests when building, you need to update your Dockerfile. Create a new test stage that runs the tests.
+
+The following is the updated Dockerfile.
+
+```dockerfile
+# syntax=docker/dockerfile:1
+
+FROM composer:lts as prod-deps
+WORKDIR /app
+RUN --mount=type=bind,source=./composer.json,target=composer.json \
+    --mount=type=bind,source=./composer.lock,target=composer.lock \
+    --mount=type=cache,target=/tmp/cache \
+    composer install --no-dev --no-interaction
+
+FROM composer:lts as dev-deps
+WORKDIR /app
+RUN --mount=type=bind,source=./composer.json,target=composer.json \
+    --mount=type=bind,source=./composer.lock,target=composer.lock \
+    --mount=type=cache,target=/tmp/cache \
+    composer install --no-interaction
+
+FROM php:8.2-apache as base
+RUN apt-get update && apt-get install -y libpq-dev && docker-php-ext-install pdo pdo_pgsql
```

Comment:
You might want to go ahead and clear the apt cache (`rm -rf /var/lib/apt/lists/*`) to ensure we're aligning with [our own best practices](https://docs.docker.com/develop/develop-images/instructions/#apt-get).